crates/ironrdp-client/src/config.rs

```diff
@@ -168,7 +169,7 @@ struct Args {
     password: Option<String>,
 
     /// Specify the security protocols to use
-    #[clap(long, value_enum, value_parser, default_value_t = SecurityProtocol::Hybrid)]
+    #[clap(long, value_enum, value_parser, default_value_t = SecurityProtocol::Ssl)]
```

Comment:
The only request I have before we actually merge, is to leave this at `Hybrid` by default. I believe the intention is to use CredSSP by default.

Reply:
Ah yeah good catch, that was just for testing I didn't mean to push it.